src/avatar/README.md

```diff
@@ -0,0 +1,47 @@
+# Avatar
+
+[View prototype on CodePen][codepen].
```

Comment:
Should all components have this sort of prototype?

Reply:
This is the first time I’ve done it, which mostly was driven by a way to actually show the components rendered. Without fleshing out a full documentation site, CodePen links felt like a cheap first stab.

What are you thoughts on it?

Reply:
I think it's great – though we may want to use the thoughtbot Codepen account for maintainability.

Reply:
Definitely, good catch! I will move this component over to the thoughtbot account and use that going forward.

Reply:
Update to use thoughtbot CodePen account.